lib/src/provider/read_provider.dart

```diff
@@ -1,5 +1,9 @@
+import 'package:starknet/src/provider/model/get_block_txn_count.dart';
 import 'package:starknet/starknet.dart';
 
+import 'model/get_block_txn_count.dart';
+import 'model/get_block_with_txs.dart';
```

Comment:
don't use relative import, everything should be accessible from "package:starknet/starknet.dart"